configure.ac

```diff
@@ -0,0 +1,72 @@
+dnl require autoconf 2.60 (AS_ECHO/AS_ECHO_N)
+AC_PREREQ([2.60])
+define(_CLIENT_VERSION_MAJOR, 0)
+define(_CLIENT_VERSION_MINOR, 93)
+define(_CLIENT_VERSION_BUGFIX, 0)
```

Comment:
Do these numbers have any effect on the build system or on Armory? I'm guessing not since Armory does its own thing. That said, it might not be a bad idea to look into how to unify everything.